src/cascadia/TerminalApp/AdminWarningPlaceholder.cpp

```diff
@@ -37,4 +38,37 @@ namespace winrt::TerminalApp::implementation
     {
         return _control;
     }
+
+    // Method Description:
+    // - Move the focus to the cancel button by default. This has the LOAD
+    //   BEARING side effect of also triggering Narrator to read out the
+    //   contents of the dialog. It's unclear why doing this works, but it does.
+    // - Using a LayoutUpdated event to trigger the focus change when we're
+    //   added to the UI tree did not seem to work.
+    // - Whoever is adding us to the UI tree is responsible for calling this!
+    // Arguments:
+    // - <none>
+    // Return Value:
+    // - <none>

```

Comment:
Is there any possible way we could write a test to verify that this is happening? Like ensure that the event that's being triggered continues to happen with a UIA test that launches the warning? (Follow on issue?)